common/app/com/commercetools/sunrise/framework/viewmodels/content/PageContent.java

```diff
@@ -36,4 +38,13 @@ public void addComponent(final ViewModelComponent component) {
         }
         this.components.add(component);
     }
+
+    public WishlistViewModel getWishlist() {
+        return wishlist;
+    }
+
+    public void setWishlist(final WishlistViewModel wishlist) {
+        this.wishlist = wishlist;
+    }
```

Comment:
Using `extendedViewModel` worked fine and as we discussed - should be fine for now.